src/bottom-bar/accept-button/accept-button.js

```diff
@@ -0,0 +1,36 @@
+import { useAlert } from '@dhis2/app-runtime'
+import i18n from '@dhis2/d2-i18n'
+import { Button } from '@dhis2/ui'
+import React, { useEffect } from 'react'
+import { useWorkflowContext } from '../../workflow-context/index.js'
+import { useAcceptData } from './use-accept-data.js'
+
+const AcceptButton = () => {
+    const [acceptData, { loading, error }] = useAcceptData()
+    const { params, refresh } = useWorkflowContext()
+    const { show } = useAlert(
+        i18n.t('Acceptance failed: {{error}}', {
```

Comment:
In order to use a colon in an `i18n` string, `nsSeparator: null` must be passed as an option to `i18n.t` as the colon is the default namespace separator. This means that `i18n` interprets this string as part of a namespace with key ` {{error}}`, resulting in the wrong output to `i18n/en.pot`.